src/Quaternion_TEST.cc

```diff
@@ -212,30 +212,53 @@ TEST(QuaternionTest, Integrate)
 }
 
 /////////////////////////////////////////////////
-TEST(QuaternionTest, Math)
+TEST(QuaternionTest, MathLog)
 {
   math::Quaterniond q(IGN_PI*0.1, IGN_PI*0.5, IGN_PI);
   EXPECT_TRUE(q == math::Quaterniond(0.110616, -0.698401, 0.110616, 0.698401));
 
   EXPECT_TRUE(q.Log() ==
       math::Quaterniond(0, -1.02593, 0.162491, 1.02593));
 
-  EXPECT_TRUE(q.Exp() ==
-      math::Quaterniond(0.545456, -0.588972, 0.093284, 0.588972));
-
   math::Quaterniond q1 = q;
   q1.W(2.0);
   EXPECT_TRUE(q1.Log() ==
       math::Quaterniond(0, -0.698401, 0.110616, 0.698401));
+}
 
+/////////////////////////////////////////////////
+TEST(QuaternionTest, MathExp)
+{
+  math::Quaterniond q(IGN_PI*0.1, IGN_PI*0.5, IGN_PI);
+  EXPECT_TRUE(q == math::Quaterniond(0.110616, -0.698401, 0.110616, 0.698401));
```

Comment:
No need to change anything in this PR, but just for future reference, it would be good to use `EXPECT_EQ(..., ...)` instead of `EXPECT_TRUE(... == ...)` for new additions. This way, when the test failures, the expected and returned values are printed, so it's easier to debug.

Reply:
I'll change in the next PR